src/KeyedFrames.jl

```diff
@@ -11,11 +11,13 @@ struct KeyedFrame <: AbstractDataFrame
 
     function KeyedFrame(df::DataFrame, key::Vector{<:Symbol})
         key = unique(key)
+        df_names = names(df)
 
-        if !issubset(key, names(df))
+        if !issubset(key, df_names)
             throw(
                 ArgumentError(
-                    "The columns provided for the key must all be present in the DataFrame"
+                    """The columns provided for the key ($key) must all be
+                    present in the DataFrame ($df_names)."""
```

Comment:
Do we want the error to be on two separate lines?. Maybe you want to just use `string` or `*`?

Reply:
I kind of like multiline errors when they're long, but I'm not bothered either way.